templates/partials/search_result_list.html

```diff
@@ -0,0 +1,21 @@
+{{ define "search_result_list" }}
+
+<div class="search-result-list">
+  <dt class="search-result-list__title">
+    {{ .title }}
+  </dt>
+  {{ if .value }}
+    <ul class="search-result-list__value divide-y divide-gray-200">
+      {{ range $v := .value }}
+        <li class="search-result-list__item">
+          <img class="search-result-list__icon" src="/dist/svg/link.svg">
```

Comment:
Same comment as above, prefer using CSS instead of inlining images. Inlining SVGS should be reserved for cases when it's necessary to override the path colours or strokes based on CSS classes. In most other cases, prefer using a background image.

Reply:
Decide to do this in another PR as I found some issue with ParcelJS
issue link: https://github.com/gutakk/go-google-scraper/issues/41